packages/mongodb-chatbot-ui/src/Chatbot.tsx

```diff
@@ -8,26 +8,41 @@ import { type User } from "./useUser";
 import { ChatbotProvider } from "./ChatbotProvider";
 
 export type ChatbotProps = {
+  children: React.ReactElement | React.ReactElement[];
   darkMode?: boolean;
-  tck?: string;
+  isExperimental?: boolean;
+  maxInputCharacters?: number;
+  name?: string;
```

Comment:
why have name here instead of on the FloatingActionButtonTrigger and InputBarTrigger?

Reply:
This is for set it once and it applies globally. You can use custom text/override the name individually on those components too.